cdk/database/__init__.py

```diff
@@ -81,6 +81,8 @@ def dynamodb_visits_table(self):
                                                sort_key=aws_dynamodb.Attribute(
                                                    name='visit_time',
                                                    type=aws_dynamodb.AttributeType.STRING))
+       
+        self.export_value(self.visits_table.table_name) #! remove in next pr
```

Comment:
What does this do?

Reply:
When I swap over the lambda to use the new table, the visitor stack still thinks it needs the export. So this explicit export holds the export until the visitor stack can be updated. This will be removed in the next pr, and then the old visits table can be removed cleanly in another pr.

Funny enough, this is an "official" workaround: https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.Stack.html#deployment-1-break-the-relationship